example-usage/ci-app-build-script/build-application.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+cd "$(dirname $0)"
+
+
+build_date="$(date +%s)"
+
+# Create a label for use during cleanup since the devcontainer CLI does 
+# not have a "remove" or "down" command yet (though this is planned).
+id_label="ci-container=${build_date}"
+
+# Run build
+devcontainer up --id-label ${id_label} --workspace-folder ../workspace
+set +e
+devcontainer exec --id-label ${id_label} --workspace-folder ../workspace scripts/execute-app-build.sh
```

Comment:
Do we need the `--id-label` argument here? I thought the CLI should be able to find the container based on the workspace folder. If so, it might be nice to remove the argument from this example to keep the execution simpler so folks don't think they have to pass the label when exec'ing.

Reply:
@jkeech Yes.  As it explains above, there's no "down" yet. So there's no other way to identify the container and kill it yet.

Reply:
I'm not talking about the `docker rm` below -- I think it's fine to do the `up` with a specific label in order to delete as the third step. This comment here is for the middle `exec` step. Can't we still `exec` using the workspace folder to find the previous container?

Reply:
@jkeech No because it identifies the container using the `id_label`.  If this was an "additional label" that would work, but that doesn't appear to be the case.  I initially didn't have it in exec, and kept getting errors that the container couldn't be found until I realized what was going on.

Might indicate we want a separate `--label` property independent of this one though since I agree that would be better.

Reply:
Got it. I thought we always added the workspace folder label. No problem if it's currently required. I think we should improve this in the CLI though which would make the sample cleaner (in addition to supporting the `down` command).